nextjs-task-system/app/admin/page.tsx

```
@@ -0,0 +1,180 @@
+          completed: 0,
+        },
+        byPriority: {
+          low: 0,
+          medium: 0,
+          high: 0,
+        },
+      };
+
+      tasks.forEach((task) => {
+        // Count by status
+        if (summary.byStatus.hasOwnProperty(task.status)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)